pageserver/src/tenant/timeline.rs

```diff
@@ -2878,6 +2877,14 @@ impl Timeline {
             ancestor_gc_info.insert_child(timeline_id, metadata.ancestor_lsn(), is_offloaded);
         }
 
+        let relsize_snapshot_cache_capacity = {
+            let loaded_tenant_conf = tenant_conf.load();
+            loaded_tenant_conf
+                .tenant_conf
+                .relsize_snapshot_cache_capacity
+                .unwrap_or(conf.default_tenant_conf.relsize_snapshot_cache_capacity)
+        };
```

Comment:
nit: This is duplicated in a bunch of places. You can abstract to `Timeline::get_relsize_snapshot_cache_capacity`.
See `Timeline::get_lsn_lease_length` for an example.

Reply:
It is used exactly in *one* place and it is not possible to used function like `get_lsn_lease_length` because it is used in Timeline struct component initialisation, so `self` is not available.

Reply:
Yep, my bad. Though it was multiple places.